product_docs/docs/biganimal/release/overview/03_security/index.mdx

```diff
@@ -16,3 +16,5 @@ BigAnimal runs in your own cloud account, isolates your data from other users, a
     -   **pgAudit:** The classes of statements being logged for pgAudit are set globally on a cluster with `pgaudit.log = 'write,ddl'`. The following statements made on tables are logged by default when the cluster type is PostgreSQL: `INSERT`, `UPDATE`, `DELETE`, `TRUNCATE`, AND `COPY`. All `DDL` is logged. 
 
 -   **Database cluster permissions** The edb_admin account created during the *create cluster* process includes the `CREATEDB` and `CREATEROLE` database roles. EDB recommends using the edb_admin account to create a new application user and new application database for further isolation. See [Managing Postgres access](../../using_cluster/01_postgres_access/) for more information.
+
+For information on BigAnimal security compliance and certifications, see [Security compliance and certifications](compliance_and_certifications/). 
```

Comment:
Another approach we could use to avoid the redundancy in this sentence is to have a See also section or intro sentence. For example,

## See also

[Security compliance and certifications](compliance_and_certifications/)